pkg/resources/containers/container.go

```diff
@@ -0,0 +1,89 @@
+package containers
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+
+	//"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"os"
+)
+
+const (
+	TimeInSeconds = 5
+)
+
+// Go build-time variables
+var ImageRepository string
+
+//func MakeContainer
+func MakeContainer(customResourceName string, version string, envVarArray []corev1.EnvVar) corev1.Container {
```

Comment:
May be renamed to `NewContainer` for consistency.

Reply:
rename to NewContainer